Assets/Scripts/TakeDamageFromPlayerProjectile.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using System.Linq;
+
+public class TakeDamageFromPlayerProjectile : MonoBehaviour
+{
+    public float health = 25.0f;
+
+    void OnCollisionEnter(Collision collision)
+    {
+        if (collision.gameObject.CompareTag(Tags.PLAYER_PROJECTILE))
+        {
+            float force = collision.relativeVelocity.magnitude;
+            health -= force;
+
+            if (health <= 0.0f)
```

Comment:
nit: Compare with `0` instead of `0.0f` (just for less verbosity)

```suggestion
            if (health <= 0)
```